single_tech_samples/databricks/README.md

```diff
@@ -1,25 +1,26 @@
+[![GitHub license](Common_Assets/Images/MIT.svg)](https://github.com/Azure-Samples/modern-data-warehouse-dataops/blob/master/LICENSE)
+[![Open Source? Yes!](Common_Assets/Images/Open_Source.svg)](https://opensource.microsoft.com/codeofconduct/)
+
+
 Azure Databricks is a data analytics platform optimized for the Microsoft Azure cloud services platform which lets you set up your Apache Spark™ environment in minutes, and enable you to autoscale, and collaborate on shared projects in an interactive workspace.
 
 The latest documentation on this services can be found [here](https://docs.microsoft.com/en-us/azure/databricks/)
 
-## Summary
+## Samples
 
-This folder captures sample implementations for the following processes around Azure Databricks:
+Following is the list of samples associated with automating the deployment of Azure Databricks Service:
 
-1. [Deploying a secure Azure Databricks cluster on Microsoft Azure](Cluster_Deployment/README.md)
-1. [Setting up a Continuous Integration pipeline](Continuous_Integration/README.md)
-1. [Setting up a Continuous Delivery pipeline](Continuous_Delivery/README.md)
+1. [Basic deployment of an Azure Databricks environment using ARM templates](sample1_basic_azure_databricks_environment/README.md)
+2. [Enterprise grade deployment of Azure Databricks environment aligning to Data Exfiltration Protection Reference Architecture](sample2_enterprise_azure_databricks_environment/README.md)
+3. [Cluster provisioning and enabling data access on a pre-provisioned Azure Databricks Workspace](sample3_cluster_provisioning_and_data_access/README.md)
 
 ----------------------- ------------------------------------
-**_NOTE:_**  These samples are built using a selection of programing languages and services which is well adopted by the enterprise customers. Developers can leverage other technologies to achieve the same objective. Please refer to the [documentation]("https://docs.microsoft.com/en-us/azure/databricks/")
- for further details on the supported technologies.
- ----------------------- ------------------------------------
+**_NOTE:_**  These samples are built using a selection of programing languages and services which is well adopted by the enterprise customers. Developers can leverage other technologies to achieve the same objective. Please refer to the [documentation]("https://docs.microsoft.com/en-us/azure/databricks/") for further details on the supported technologies.
+----------------------- ------------------------------------
 
 ## Contributing
 
-### Markdown linting
-
-Please run `make lint` before submitting the PR to make sure the links in your Markdown files
-are alive.
\ No newline at end of file
+Please refer to the [contribution guide](https://github.com/Azure-Samples/modern-data-warehouse-dataops) published at the root level of this repository.
```

Comment:
My bad -- we don't have a public contribution guide just yet. The only one we have is MS internal only. But yes, it would good to have a public one as well. 
For now, can we take this out just to avoid confusion (ei. looking for a contribution guide that isn't there).

Reply:
Fixed